src/main/java/seedu/address/logic/parser/LinkCommandParser.java

```diff
@@ -16,17 +19,19 @@ public class LinkCommandParser implements Parser<LinkCommand> {
      * @throws ParseException if the user input does not conform the expected format
      */
     public LinkCommand parse(String args) throws ParseException {
-        Boolean isEmpty = args.trim().isEmpty();
-        Boolean isInvalid = !args.trim().matches("\\d+(\\s+\\d+)*");
+        String trimmed = args.trim();
+        Boolean isEmpty = trimmed.isEmpty();
+        String validationRegex = "[1-9][0-9]*(\\s+[1-9][0-9]*)*";
```

Comment:
Not too sure about this. What if the input is 02? It should be ok right?

Reply:
02 is not a valid input in normal cases. we can specify that in our UG.

Reply:
It is a valid input as index in commands like `delete` or `edit`. There is a parser class method for parsing index, I think you can use that.